middleware/checkExistence.js

```diff
@@ -0,0 +1,21 @@
+const { responseHandler } = require('./handlers');
+const { User } = require('../models');
+
+const checkExistence = async (req, res, next) => {
+  const { username } = req.body;
+
+  const user = await User
+  .findOne({
+    where: { username },
+  })
+
+  if (user !== null) {
```

Comment:
I think changing it to `if(user) `might be better than `if(!user)` since the middleware is checking to see if the user exists and if so then the server should respond 'username already exists, please choose another one'.

Reply:
yeah, mixed that up...your login middleware should be the one with `if(!user)` ..is there a route for your login on this PR, didn't see the one

Reply:
Yup lol it just clicked that's why I deleted the comment